backend/app/services/flow.py

```
@@ -0,0 +1,1646 @@
+            .offset(skip)
+            .limit(limit)
+            .all()
+        )
+
+        if not executions:
+            return [], total
+
+        # Batch load all related flows (fixes N+1 query issue)
+        flow_ids = list(set(e.flow_id for e in executions))
+        flows = db.query(FlowResource).filter(FlowResource.id.in_(flow_ids)).all()
+        flow_map = {f.id: f for f in flows}
```

Ruff (0.14.13)

Local variable `flow_map` is assigned to but never used

Remove assignment to unused variable `flow_map`

(F841)